cmd/status.go

```diff
@@ -0,0 +1,308 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/fatih/color"
+	"github.com/mattn/go-isatty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+var (
+	// taken from https://github.com/fatih/color/blob/4d2835ff85a014514ee435d49f76dc8b25c9cee3/color.go#L20-L21
+	noColor = os.Getenv("TERM") == "dumb" ||
+		(!isatty.IsTerminal(os.Stdout.Fd()) && !isatty.IsCygwinTerminal(os.Stdout.Fd()))
+)
+
+const (
+	productionURL = "https://epr.elastic.co"
```

Comment:
I have few concerns about this file. I have a feeling that it should be split into multiple units. What do you think about the following approach:
1. Leave the table rendering and package root selection in `cmd`.
2. Create new package for Package Registry client like we have ones for `elasticsearch` and `kibana`.
3. Create new package for status (e.g. `internal/packages/status`) that can provide package deployment status for the given package (based on the package root).